tests/fixtures/webext-as-library/helpers.js

```diff
@@ -0,0 +1,15 @@
+const assert = require('assert');
+
+function testWebExtModuleImports(webExt) {
+  assert.deepEqual(Object.keys(webExt).sort(), ['cmd', 'main', 'util'].sort());
+  assert.deepEqual(Object.keys(webExt.util).sort(), ['logger', 'adb'].sort());
+  assert.deepEqual(
+    Object.keys(webExt.util.adb).sort(),
+    ['listADBDevices', 'listADBFirefoxAPKs'].sort(),
+  );
+  assert.equal(typeof webExt.cmd.run, 'function');
```

Comment:
Nit, I would move this at line 6 (just to "visually" keep the assertions related and the ones unrelated with adb in their separate group of assertions)